analysis/druidrestoration/src/modules/features/WildGrowth.tsx

```diff
@@ -1,32 +1,28 @@
 import { t } from '@lingui/macro';
 import { formatPercentage } from 'common/format';
 import SPELLS from 'common/SPELLS';
-import { SpellLink } from 'interface';
-import { SpellIcon } from 'interface';
+import { SpellIcon, SpellLink } from 'interface';
 import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
-import Events, { AnyEvent, ApplyBuffEvent, CastEvent, HealEvent } from 'parser/core/Events';
+import Events, { AnyEvent, CastEvent, EventType, HealEvent } from 'parser/core/Events';
 import { ThresholdStyle, When } from 'parser/core/ParseResults';
 import AbilityTracker from 'parser/shared/modules/AbilityTracker';
 import HealingValue from 'parser/shared/modules/HealingValue';
 import BoringValue from 'parser/ui/BoringValueText';
+import { PerformanceBoxRow } from 'parser/ui/PerformanceBoxRow';
 import Statistic from 'parser/ui/Statistic';
 import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 
+import { getHeals } from '../../normalizers/CastLinkNormalizer';
+
 /** Number of targets WG must effectively heal in order to be efficient */
 const RECOMMENDED_EFFECTIVE_TARGETS_THRESHOLD = 3;
-/** Max buffer in ms from WG cast to WG apply */
-const APPLY_BUFFER = 100;
 /** Max time after WG apply to watch for high overhealing */
 const OVERHEAL_BUFFER = 3000;
 /** Overheal percent within OVERHEAL_BUFFER of application that will count as 'too much' */
 const OVERHEAL_THRESHOLD = 0.5;
 
 /**
  * Tracks stats relating to Wild Growth
- *
- * This modules functioning relies on normalizers that ensure:
- * * Wild Growth cast will always come before Wild Growth HoT apply
- * * Wild Growth HoT apply will always come before Wild Growth first heal
  */
 class WildGrowth extends Analyzer {
```

Comment:
Refactored this to use `EventLinkNormalizer` (the new way I've been doing things) instead of event order tracking